src/inspector/main_thread_interface.cc

```diff
@@ -150,8 +150,8 @@ class MainThreadSessionState {
 
   static std::unique_ptr<MainThreadSessionState> Create(
       MainThreadInterface* thread, bool prevent_shutdown) {
-    return std::unique_ptr<MainThreadSessionState>(
-        new MainThreadSessionState(thread, prevent_shutdown));
+    return std::make_unique<MainThreadSessionState>(
+        thread, prevent_shutdown);
```

Comment:
This fits on one line, doesn't it?

Reply:
Yes. Looks like I really need set up my clang-format right in WSL ...